examples/tv-app/android/java/MediaPlaybackManager.cpp

```diff
@@ -220,6 +220,38 @@ uint64_t MediaPlaybackManager::HandleMediaRequestGetAttribute(MediaPlaybackReque
     return ret;
 }
 
+long MediaPlaybackManager::HandleMediaRequestGetLongAttribute(MediaPlaybackRequestAttribute attribute)
```

Comment:
A doc comment saying `returns 0 on error` may help.

Reply:
I will add this in a subsequent PR, thanks